spec/models/instrument_price_policy_calculations_spec.rb

```diff
@@ -16,18 +17,58 @@
     policy.calculate_cost_and_subsidy_from_order_detail fake_order_detail
   end
 
-  it "calculates cost based on the given duration and discount" do
-    policy.usage_rate = 5
-    expect(policy.calculate_cost(120, 0.15).round 2).to eq 1.5
+  describe "estimating cost and subsidy from an order detail" do
```

Comment:
These specs read a lot more like feature specs than unit tests, and to be honest it's hard for me to tell if they are testing all the right things. I gather most of the changes in this file are simple rearrangements, and it may not be worth the effort of re-writing these at this point, but thought I would flag it in case we end up doing more refactoring around price policies.